docs/src/user-guide/command-line-interface.md

```diff
@@ -38,99 +38,43 @@ npx eslint "lib/**"
 
 **Note:** You can also use alternative package managers such as [Yarn](https://yarnpkg.com/) or [pnpm](https://pnpm.io/) to run ESLint. Please refer to your package manager's documentation for the correct syntax.
 
-## Options
-
-The command line utility has several options. You can view the options by running `npx eslint -h`.
-
-```text
-eslint [options] file.js [file.js] [dir]
-
-Basic configuration:
-  --no-eslintrc                   Disable use of configuration from .eslintrc.*
-  -c, --config path::String       Use this configuration, overriding .eslintrc.* config options if present
-  --env [String]                  Specify environments
-  --ext [String]                  Specify JavaScript file extensions
-  --global [String]               Define global variables
-  --parser String                 Specify the parser to be used
-  --parser-options Object         Specify parser options
-  --resolve-plugins-relative-to path::String  A folder where plugins should be resolved from, CWD by default
```

Comment:
I'm not sure if we want to remove this overview of all options.

(this is the output from `eslint -h`)

Reply:
i removed b/c user can just run the command to get this. i don't think there's value in documenting the output. also if the command this would also have to changed or become out of date. 

i think having it provides a non-trivial maintenance burden for little value.

Reply:
I also feel like we can have this

Reply:
added back w refactored intro text in [78affe7](https://github.com/eslint/eslint/pull/16563/commits/78affe73a22805015a9af1a16b1ab6e755a8f616)